cortx-motr.spec.in

```diff
@@ -81,10 +97,15 @@ BuildRequires:  make
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
 %if %{rhel} <= 7
+%if "%{?CONFIG_ARCH_TYPE}" == "x86_64"
 BuildRequires:  gccxml
+%else if "%{?CONFIG_ARCH_TYPE}" == "aarch64"
+#BuildRequires:  castxml
```

Comment:
is castxml or gccxml is not required for ARM arch ?

Reply:
castxml pkg is not available for  centos 7.9 version, we are manfully setting the castxml binary

Reply:
Can you add a todo here, once castxml is available this needs to be updated ?